python/src/autogenstudio/datamodel/db.py

```diff
@@ -65,6 +65,25 @@ class Message(BaseDBModel, table=True):
     run_id: Optional[int] = Field(default=None, sa_column=Column(Integer, ForeignKey("run.id", ondelete="CASCADE")))
 
     message_meta: Optional[Union[MessageMeta, dict]] = Field(default={}, sa_column=Column(JSON))
+    feedbacks: List["Feedback"] = Relationship(back_populates="message")
```

Comment:
nit (plural doesn't have an "s"): 
```suggestion
    feedback: List["Feedback"] = Relationship(back_populates="message")
```